ntoskrnl/ke/i386/kiinit.c

```diff
@@ -85,58 +85,6 @@ KiInitMachineDependent(VOID)
     /* Check for PAT support and enable it */
     if (KeFeatureBits & KF_PAT) KiInitializePAT();
 
-    /* Assume no errata for now */
-    SharedUserData->ProcessorFeatures[PF_FLOATING_POINT_PRECISION_ERRATA] = 0;
-
-    /* Check if we have an NPX */
-    if (KeI386NpxPresent)
```

Comment:
Since you now assume always we support CPUs with an **NPX** :P (aka math coprocessor aka FPU), that FPU errata seems to be important to be controlled.

Reply:
That is what `KiIsNpxErrataPresent` function is used for

Reply:
Ah ok, so my understanding correct, that in a sense, that loop done below (and deleted) is in a way "done", now when each CPU is being initialized separately and KiVerifyCpuFeatures() is called?

Reply:
Yes. `KiVerifyCpuFeatures` is called on each CPU, so we verify all of them don't have that FPU bug